src/config/ConfigurationDefault.h

```diff
@@ -9,13 +9,13 @@ namespace config {
 
 const std::vector<Setting> kDefaultConfig = {
     { ConfigurationName::kAngleSensorMaxValue,
-        M_PIf,
+        M_PI,
```

Comment:
A simple matter of float vs double precision?

Reply:
Exactly, but I will fix the reading of config to not only depend on `double`